Inspicio/wwwroot/js/multipleImages.js

```diff
@@ -10,30 +10,35 @@
         var description = $("#Image_Description_UserInput").val();
         var content = $("#b64uploader").val();
         if (title.length !== 0 && content.length !== 0) {
-
-            $("#uploader").css("width", "100%");
-            $(this).removeClass("ready");
-            var contentCol = listOfImages.map(v => v.Content);
-            if ($.inArray(content, contentCol) > -1) {
-             alert("This image is already added");
-             $("#uploader").html("").append(dropzoneMainText);
-             $(dropzoneMainText).show();
+            if (title.length < 5) {
+                alert("Screen title must have at least 5 characters!")
```

Comment:
Nah, it's not that really that important, might as well leave it - I was just wondering. I'd assumed that there wouldn't be a limit - so an empty string would be fine for image title and screen title. Just wondering what made you decide to require at least 5 chars is all.